src/buildblock/ProjDataFromStream.cxx

```diff
@@ -194,6 +194,96 @@ ProjDataFromStream::get_viewgram(const int view_num, const int segment_num,
   return viewgram;    
 }
 
+float
+ProjDataFromStream::get_bin_value(const Bin& this_bin) const
+{
+    return get_bin_value(this_bin.segment_num(),
+                         this_bin.axial_pos_num(),
+                         this_bin.view_num(),
+                         this_bin.tangential_pos_num());
+}
+
+
+float
+ProjDataFromStream::get_bin_value(const int segment_num,
+                                  const int axial_pos_num,
+                                  const int view_num,
+                                  const int tang_pos_num) const
+{
+    if (sino_stream == 0)
+    {
+        error("ProjDataFromStream::get_viewgram: stream ptr is 0\n");
+    }
+    if (! *sino_stream)
+    {
+        error("ProjDataFromStream::get_viewgram: error in stream state before reading\n");
```

Comment:
error message says "get_viewgram".

Also, delete \n from end of error messages (also a few lines above)